lib/views/backend_v1.2/spree/admin/payments/source_forms/_paypal_braintree.html.erb

```diff
@@ -0,0 +1,16 @@
+<% id = payment_method.id %>
+
+<fieldset class="no-border-bottom braintree-hosted-fields" data-braintree-hosted-fields data-id="braintree" data-payment-method-id="<%= id %>">
```

Comment:
@Sinetheta like discussed earlier we don't want to have Solidus version comparison in view files. This file will be gone soon as we drop support for Solidus < 1.2

We could move all the parts that we share into a partial and only leave the wrapper in these two partials, but again, this is just an intermediate solution.